translations/de.i18n.json

```diff
@@ -1,9 +1,11 @@
 {
   "de": {
     "moderation": {
-      "report_button": "Bericht",
+      "report_button": "Senden",
+      "report_content": "Melden",
```

Comment:
so report_content is missing in every language except english ? can you check with product if we already have a ticket to translate this text, and if there isn't can you open one please ?
thanks

Reply:
will do